models/public/higher-hrnet-w32-human-pose-estimation/higher-hrnet-w32-human-pose-estimation.md

```diff
@@ -0,0 +1,86 @@
+# higher-hrnet-w32-human-pose-estimation
+
+## Use Case and High-Level Description
+
+The `HigherHRNet-W32` model is one of the [HigherHRNet](https://arxiv.org/pdf/1908.10357).
+`HigherHRNet` is a novel bottom-up human pose
+estimation method for learning scale-aware representations using high-resolution feature pyramids. The network uses HRNet as backbone, followed by one or more deconvolution modules to generate multi-resolution and high-resolution heatmaps. For every person in an image, the network detects a human pose: a body skeleton consisting of keypoints and connections between them. The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wrists, hips, knees, and ankles.
+This is PyTorch\* implementation pre-trained on COCO dataset.
+For details about implementation of model, check out the [HigherHRNet: Scale-Aware Representation Learning for Bottom-Up Human Pose Estimation](https://github.com/HRNet/HigherHRNet-Human-Pose-Estimation) repository.
+
+## Specification
+
+| Metric            | Value                  |
+|-------------------|------------------------|
+| Type              | Human pose estimation  |
+| GFLOPs            | 92.8364                |
+| MParams           | 28.6180                |
+| Source framework  | PyTorch\*              |
+
+## Accuracy
+
+| Metric                     | Original model    | Converted model |
+| -------------------------- | ----------------- | --------------- |
+| Average Precision (AP)     | 64.63%            | 64.63%          |
```

Comment:
Is it specified somewhere in this readme file, what dataset and split was used to collect these metrics? Or the user is supposed to go look it in the accuracy checker config?

Reply:
Also it might be a bit confusing whether "original" means the model and the code from original repo, or the model in original framework tested in Accuracy Checker with its own implementation of a decoding stage. But if this notation is used throughout the whole OMZ having the same meaning, then it's absolutely fine.

Reply:
Dataset is specified in the section `Use Case and High-Level Description`. As I know the purpose of OMZ is to check whether the accuracy has decreased during the conversion of the model. In my opinion models should be tested under the same conditions. Another thing is that we changed the model to make it similar in postprocessing with other `ae` models.

Reply:
> Dataset is specified in the section Use Case and High-Level Description

It's said, that the model is "pre-trained on COCO dataset". One may guess, that it is tested on COCO dataset as well. But what split is used then? val2017 or test-dev2017?

> As I know the purpose of OMZ is to check whether the accuracy has decreased during the conversion of the model.

This is 100% fine and up to you. I'm only talking about how clear the documentation is. As long as the comment is minor, you may ignore it.

Reply:
Documentation updated